dev-qt/libqt4pas/libqt4pas-2.5.0.recipe

```diff
@@ -0,0 +1,71 @@
+SUMMARY="Pascal Qt interface"
+DESCRIPTION="Pascal wrapper around Qt interfaces.
+Just the necessary stuff for Lazarus is wrapped here."
+HOMEPAGE="http://users.telenet.be/Jan.Van.hijfte/qtforfpc/fpcqt4.html"
+SOURCE_URI="http://users.telenet.be/Jan.Van.hijfte/qtforfpc/V2.5/qt4pas-V2.5_Qt4.5.3.tar.gz"
+CHECKSUM_SHA256="825423db80da4df5c21816c0392b3394cddfe2f3293dfd08ace84941726affea"
+SOURCE_DIR="qt4pas-V2.5_Qt4.5.3"
+LICENSE="GNU LGPL v2.1"
```

Comment:
The correct [order](https://github.com/haikuports/haikuports/wiki/HaikuPorter-Guidelines#ordering), after DESCRIPTION, is:
```
HOMEPAGE="http://users.telenet.be/Jan.Van.hijfte/qtforfpc/fpcqt4.html"
COPYRIGHT="2005-2011 Jan Van hijfte"
LICENSE="GNU LGPL v2.1"
REVISION="1"
SOURCE_URI="http://users.telenet.be/Jan.Van.hijfte/qtforfpc/V2.5/qt4pas-V2.5_Qt4.5.3.tar.gz"
CHECKSUM_SHA256="825423db80da4df5c21816c0392b3394cddfe2f3293dfd08ace84941726affea"
SOURCE_DIR="qt4pas-V2.5_Qt4.5.3"

ARCHITECTURES="!x86_gcc2 x86 x86_64"
SECONDARY_ARCHITECTURES="x86"
```